File: app/src/main/java/com/calldoradosdk/integration/SetDeleteUserDataReceiver.java
package com.calldoradosdk.integration;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.calldorado.sdk.Calldorado;
import com.calldorado.sdk.util.Constants;

public class SetDeleteUserDataReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if (intent.getAction() == Constants.CDO_DELETE_MY_DATA_INTENT) {
            //Stop your own third parties here if you need to
            Calldorado.deleteAppData();
        }
    }
}

File: app/src/main/java/com/calldoradosdk/integration/SetDeleteUserDataReceiver.kt
package com.calldoradosdk.integration

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.calldorado.sdk.Calldorado.deleteAppData
import com.calldorado.sdk.util.Constants

class SetDeleteUserDataReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action === Constants.CDO_DELETE_MY_DATA_INTENT) {
            //Stop your own third parties here if you need to
            deleteAppData()
        }
    }
}